app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] I think we need the `new` action in this controller. Based on the requirements of this project, there should be a form in the views/comments/new.html.erb file which accepts inputs for creating comments. You did a great job :+1: by creating a partial in this path(views/shared/_add_comment.html.erb) for adding comments. Hence the partial should be rendered in this file.

Reply:
![image](https://user-images.githubusercontent.com/106561762/205330004-038f363c-4ce6-453d-b89e-9c56a0bb0457.png)

Thanks for taking it up with me on Slack, my friend.

I'll add the "new" method in the comment controller as you recommended. (Although it is not a requirement of the project but to improve the organization)